### PR TITLE
Hypothesis Results now summarized on completion

### PR DESCRIPTION
**Pull request by Evan-CTL**

### CODE REVIEW

File: media/scss/_listset.scss [hunk 1]
```diff
@@ -10,18 +10,6 @@
     &-listnum li {
         counter-increment: listNum-counter;
     }
-    &-listnum li::before {
```
Comment (zmustapha) [inline media/scss/_listset.scss:13]: The removal of this shows weird CSS on my dev env

Comment (Evan-CTL) [reply media/scss/_listset.scss]: I've restored the CSS and fixed the broken highlight.

**Comment by zmustapha** [comment]
Information displayed is good. I'll apply UI treatment once everything is there. I'm tagging Natalia as reviewer for code.

**Review by ndittren — CHANGES_REQUESTED** [review]
I left a comment above. But also generally there is a lot happening in lines 62-102. I have to think about this restructuring a little more. Will put this PR on hold.

File: media/js/src/simulationOne/quiz.jsx [hunk 2]
```diff
@@ -577,7 +578,26 @@ export const Quiz = ({
                                     <button className=
                                         "btn btn-small btn-secondary mt-3"
                                     id="proceed"
-                                    onClick={onComplete}>
+                                    onClick={() => {
```
[the remark this answers is not in the document]
Comment (Evan-CTL) [reply media/js/src/simulationOne/quiz.jsx]: Added a handleComplete as requested

**Pull request closed without merging**